ramls/patron-action-session.raml

```diff
@@ -47,3 +48,39 @@ resourceTypes:
           exampleItem: !include examples/patron-action-session.json
       put:
         is: [validate]
+  /expired-session-patron-ids:
+      get:
+        queryParameters:
+          action_type:
+            description: Action type for patron notice session
+            type: string
+            enum: ["Check-in", "Check-out"]
+            required: true
+          last_time_action_limit:
```

Comment:
It's better to name this parameter "period_of_inactivity" and make it integer
This value is stored in minutes in Settings - Circulation - Other settings